apps/arc/little-arc.talon

```diff
@@ -0,0 +1,6 @@
+user.running: arc
+-
+little arc [<user.text>]:
```

Comment:
Maybe add a comment here that this needs to be enabled in the Arc settings to work globally.

Reply:
Thank you! I'm not sure I understood the feedback - is there some setting that you're aware of? I thought this was just enabled by default.

Reply:
Maybe it's enabled by default and I turned it off and forgot.

<img width="412" alt="Screenshot 2023-12-13 at 09 57 56" src="https://github.com/talonhub/community/assets/3219671/ca82ca1a-59cc-4e5f-b7b9-5ef9971161d1">

Reply:
Got it! Added a comment to clarify that requirement.